package.json

```diff
@@ -23,7 +24,6 @@
   "types": "dist/index.d.ts",
   "dependencies": {
     "@internetarchive/analytics-manager": "^0.1.2",
-    "@internetarchive/collection-name-cache": "^0.2.16",
```

Comment:
Q: do you think we can deprecate this at the package level?

Reply:
Absolutely! It still needs to be excised from Offshoot itself, but I believe that will be fairly simple -- it really only exists there to be passed into CB anyway.

Reply:
ok, here is the ticket for it: https://webarchive.jira.com/browse/WEBDEV-6594